### ISSUE TITLE
Use of warpSet or gaussNorm for CyTOF mass cytometry data

### ISSUE THREAD

**Issue by ashipde** [RGLab/flowStats#32]
Can one use the warpSet or gaussNorm function of flowStats to normalize high-dimensional CyTOF data? I have such data with a 36-marker panel for 70 samples and I want to reduce batch effects among the samples through normalization.

**Comment by SamGG** [comment]
I think Greg will bring a solid answer.
I just want to point you to the two recent articles on that subject and my post on their approach.
http://cytoforum.stanford.edu/viewtopic.php?f=3&t=1875
If you are interested in gaussNorm, Sofie VG has a script that she probably set up during her analysis. I didn't evaluate this as the distribution around the zero looks like a truncated gaussian in the best of the case. I think gaussNorm should be used to realign the positive peak, but the zero stays at zero.
https://github.com/saeyslab/CytoNorm_Figures/blob/master/GaussNorm.Rmd
Don't hesitate to share your feedback on Cytoforum.
Best.

**Comment by gfinak** [comment]
I agree, I recommend a more modern method. We are not really supporting gaussNorm or warpSet any more.
